lib/vcard/field.rb

```diff
@@ -241,13 +241,15 @@ def copy
       # fields that are already wrapped.
       def encode(width=nil)
         width = 75 unless width
```

Comment:
Would it work to just supply 75 as the default value in the method definition?

Reply:
I hoped so too but currently It wouldn't, because there are use cases where nil is explicitly passed.

https://github.com/qoobaa/vcard/blob/3bb579afe69f7da0479a5af3409f14dba21c1cc1/lib/vcard/dirinfo.rb#L263

Reply:
Could we change the usages instead so they don't pass an explicit nil?

Reply:
I didn't touch the line in this PR, and changing it can break user code, so can we do that separately later?

I suspect it could be intentional because DirectoryInfo#encode() delegates what is given to Field#encode().
https://github.com/qoobaa/vcard/blob/3bb579afe69f7da0479a5af3409f14dba21c1cc1/lib/vcard/dirinfo.rb#L246-L248

To fix it, DirectoryInfo#encode() needs to have the same default value or use `*args`.

Reply:
Filed https://github.com/qoobaa/vcard/pull/34.

Reply:
Thanks @knu. I've merged that.